Content.Shared/Nutrition/EntitySystems/ThirstSystem.cs

```diff
@@ -50,12 +50,12 @@ public override void Initialize()
 
     private void OnMapInit(EntityUid uid, ThirstComponent component, MapInitEvent args)
     {
-        // Do not change behavior unless starting value is explicitly defined
+        // If CurrentThirst is defined already, it uses that instead.
         if (component.CurrentThirst < 0)
         {
             component.CurrentThirst = _random.Next(
-                (int) component.ThirstThresholds[ThirstThreshold.Thirsty] + 10,
-                (int) component.ThirstThresholds[ThirstThreshold.Okay] - 1);
+                (int) component.thirstVariationMin,
+                (int) component.thirstVariationMax);
         }
```

Comment:
this really should use the thresholds or else any modification to them is going to require also updating these values for parity.